File: Chapter6/Python/Example2.py
# Chapter 6 - Κανονική Κατανομή
# Example 2: Υπολογισμός Κβαντιλίων  X ~ N(50, 10²)

from scipy.stats import norm

# Parameters
mu = 50
sigma = 10

# Quantiles
q25 = norm.ppf(0.25, loc=mu, scale=sigma)
q50 = norm.ppf(0.50, loc=mu, scale=sigma)
q75 = norm.ppf(0.75, loc=mu, scale=sigma)

print(f"First quartile (Q1): {q25:.2f}")
print(f"Median (Q2): {q50:.2f}")
print(f"Third quartile (Q3): {q75:.2f}")
print(f"Interquartile Range (IQR): {q75 - q25:.2f}")

File: Chapter6/R/Example2.R
# Chapter 6 - Κανονική Κατανομή
# Example 2: Υπολογισμός Κβαντιλίων  X ~ N(50, 10²)

# Parameters
mu <- 50
sigma <- 10

# Quantiles
q25 <- qnorm(0.25, mean = mu, sd = sigma)
q50 <- qnorm(0.50, mean = mu, sd = sigma)
q75 <- qnorm(0.75, mean = mu, sd = sigma)

cat(sprintf("First quartile (Q1): %.2f\n", q25))
cat(sprintf("Median (Q2): %.2f\n", q50))
cat(sprintf("Third quartile (Q3): %.2f\n", q75))
cat(sprintf("Interquartile Range (IQR): %.2f\n", q75 - q25))
